### ISSUE TITLE
} in comment breaks it

### ISSUE THREAD

**Issue by donnut** [leafgarland/typescript-vim#58]
If the `}` character is used in comments. it breaks it, see image
![typescript-comment-bug](https://cloud.githubusercontent.com/assets/417406/8592890/13b427ee-2633-11e5-9432-311b46091b1c.png)

**Comment by leafgarland** [comment]
thanks for the bug report.

**Comment by donnut** [comment]
Sorry, I noticed it is solved in 9a9be30. I though I had pull the latest but apparently I didn't.

BTW in line 
```
syn match typescriptLineComment "\/\/.*" contains=@Spell,typescriptCommentTodo,typescriptRef
```
the variable `typescriptRef` is never declared

**Comment by leafgarland** [comment]
Thanks again. Yes, there are a few unreferenced syntax names and highlight names in there. We really need to spend a bit of time cleaning it up. Will raise a new issue for that.
